R/utils.R

```diff
@@ -327,6 +328,19 @@ check_list_of_bool <- function(x, names, arg = caller_arg(x), call = caller_env(
   }
 }
 
+with_indexed_errors <- function(expr,
+                                message,
+                                error_call = caller_env(),
+                                envir = caller_env()) {
```

Comment:
Minor: Use `env` instead of `envir` in this sort of code?

Reply:
I was just being consistent with cli here 😞